### PR TITLE
Remove references to old styleguide

### PR DESCRIPTION
**Pull request by msecret**
Includes:
- Documentation updates
- Removing old files for kss styleguide
- Removing references to old files and folders for odl styleguide.

Fixes issue #2557 

[:sunglasses: PREVIEW](https://federalist.fr.cloud.gov/preview/18f/doi-extractives-data/remove_old_styleguide/)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -103,11 +103,7 @@ If deploying the site to a production environment, make sure to minify the JS fi
 ### Styleguide
 
 ```sh
-npm install --dev
-npm run init-styleguide
-cd styleguide-template && npm install
-cd ..
-npm run watch-styleguide
+docker-compose run webpack npm run build-styleguide
```
Comment (jeremiak) [inline CONTRIBUTING.md]: Do you want to also include some information about how to run the styleguide in watch mode instead? It seems like you'd want to be doing watch mode during dev and build would really only happen on Federalist?

File: _maps.yml [hunk 2]
```diff
@@ -4,7 +4,6 @@ exclude:
   - package.json
   - show
   - test
-  - styleguide-template
```
Comment (jeremiak) [inline _maps.yml]: This is a larger question about this `_maps.yml` file: what is it used for?

Comment (msecret) [reply _maps.yml]: I don't know, we should put it in the question doc.

**Comment by msecret** [comment]
In terms of a docker service, I think our docker build is a little slow already, so am a little worried about adding more to it. That being said, they run in parallel, so not sure perf will be changed that much by doing it.

**Comment by coreycaitlin** [comment]
@jeremiak Now that we've tackled the style guide in #2625, is this ready for final review + merge?

**Pull request merged**